pkg/resource/sensitive.go

```diff
@@ -241,7 +226,7 @@ func GetSensitiveParameters(ctx context.Context, client SecretClient, from runti
 					return errors.Wrapf(err, errFmtCannotGetSecretKeySelector, expandedJSONPath)
 				}
 				sensitive, err = client.GetSecretValue(ctx, *sel)
-				if err != nil {
+				if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
We can use [resource.IgnoreNotFound(err)](https://github.com/crossplane/crossplane-runtime/blob/428b7c3903756bb0dcf5330f40298e1fa0c34301/pkg/resource/resource.go#L207) instead.

Reply:
done